CRM/EventsExtras/Hook/BuildForm/EventInfo.php

```diff
@@ -33,6 +33,10 @@ private function buildForm($formName, &$form) {
    $this->setDefaults($form);
   }
 
+    /**
+   * Set defaults for form.
+   *
```

Comment:
Missing param description :)

Reply:
Change hook_civicrm_pre class from Event to ManageEvent as per CiviCRM Core class for event pages.